src/service/resources-impl.js

```diff
@@ -487,6 +487,7 @@ export class Resources {
       resource.requestMeasure();
       dev().fine(TAG_, 'resource reused:', resource.debugid);
     } else {
+      this.relayoutAll_ = true;
```

Comment:
I was primarily reacting to the fact that `schedulePass()` is not called here. I.e. we'd wait for the next upgrade before the new remeasuring pass is executed. Quite frankly, we can simply call here `this.schedulePass(1000);` or such, which would automatically debounce things for us.